example/src/app.js

```diff
@@ -19,6 +20,58 @@ LocaleConfig.locales['fr'] = {
 LocaleConfig.defaultLocale = 'fr';
 */
 
+LocaleConfig.locales['es'] = {
```

Comment:
I love Spanish :) but unfortunately, we can't use it as a default language in our app example.
What we can do is to add locale configuration at the main screen so the developer can use his preferred locale.
If you can contribute it it will be great, otherwise, please change it to English.